dlib/dnn/layers_abstract.h

```diff
@@ -1919,6 +1930,38 @@ namespace dlib
                 - returns the mode of this layer, either CONV_MODE or FC_MODE.  
         !*/
 
+        void disable(
+        );
+        /*!
+            ensures
+                - sets the params.size() of this layer to 0.
+                - when forward_inplace and backward_inplace are called, they return immediately.
+        !*/
+
+        alias_tensor_instance get_gamma();
+        /*!
+            ensures
+                - returns the gamma parameter that defines the behavior of forward().
```

Comment:
Update the docs for the class to talk about the parameters in terms of `gamma` and `beta`.  Or rename these methods and docs to talk in terms of A and B params.  But one way or another make it so they are in sync with how they talk about them so that someone reading the docs will know for sure what is being referenced.